pyproject.toml

```diff
@@ -39,16 +39,16 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.8"
-python-simple-logger = "^1.0.19"
-requests = "^2.31.0"
-rich = "^13.7.1"
-ipdb = "^0.13.13"
-python-rrmngmnt = "^0.1.32"
+python-simple-logger = "*"
```

Comment:
do not do this please, keep it pinned

Reply:
Just one package or all? This would prevent repos from being able to consume the latest unless this repository moves as well, no?

Reply:
Nevermind. renovate should handle it.